sdks/android/README.md

```diff
@@ -354,7 +347,46 @@ private fun checkSDKReadiness() {
 }
 ```
 
-### Track Events
+### 3. Execute AI Models
+
+Run Python workflows from your Kotlin code:
+
+```kotlin
+import dev.deliteai.datamodels.NimbleNetTensor
+import dev.deliteai.impl.common.DATATYPE
+
+private fun runPyWorkflow() {
+    val inputs = hashMapOf(
+        "input_key" to NimbleNetTensor(
+            data = "hello world",
+            datatype = DATATYPE.STRING,
+            shape = intArrayOf() // Populate only if data is an array
+        )
+    )
+
+    CoroutineScope(Dispatchers.Default).launch {
+        val result = NimbleNet.runMethod(
+            methodName = "function1",
+            inputs = inputs
+        )
```

Comment:
Let's add comment on top explaining what's happening here i.e. how's the Python function invoked and arguments get transformed.